ui/client/components/graph/node-modal/editors/expression/AceWithFeatureFlags.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable i18next/no-literal-string */
+import React, {useCallback, useMemo} from "react"
+import {useFFlags} from "../../../../../common/FeatureFlagsUtils"
+import AceWrapper, {AceWrapperProps} from "./AceWrapper"
+
+export default function AceWithFeatureFlags(props: Omit<AceWrapperProps, "noWrap" | "showLines">): JSX.Element {
```

Comment:
Are these feature flags in our current meaning? we use "feature flags" to be able to hide/show some features from user. Here they can enable them?

Reply:
maybe not. switches are hidden but accessible under hotkey

Reply:
it was part of my discussion with @lciolecki. we don't know what is the best so why not give user choice?

Reply:
In future we can create special page with all feature flags and switches. But right now it's enough.

Reply:
@JulianWielga thanks for separation, can we just rename this class to AceWithSettings?